### DISCUSSION TITLE
Is it possible to disable the line numbers in the preview window?

### DISCUSSION THREAD

**Discussion by cmvanb** [junegunn/fzf#4040] [Q&A] [answered]
![2024-10-15--00-44-02](https://github.com/user-attachments/assets/df2ea067-39ee-4065-9cf1-533f46fcfa6d)

The line numbers in the preview window (white bg, black fg '1/62' in the screenshot) are unnecessary for me, can I disable them?

**Comment by LangLangBart** [comment, marked as answer]
Unreleased feature[^1][^2]; you'll need to use the current dev version.


```sh
--preview-window noinfo
```

Alternatively: #1807

---

@junegunn, the addition might be noteworthy enough to add to the `CHANGELOG.md`?

[^1]: [github.com/junegunn/fzf/branch_commits/952276d](https://github.com/junegunn/fzf/branch_commits/952276dc2dcdd5401be804b4417d0cb392e2eda8)
[^2]: [Add 'noinfo' option to hide scroll offset information in preview window · junegunn/fzf@952276d](https://github.com/junegunn/fzf/commit/952276dc2dcdd5401be804b4417d0cb392e2eda8)

**Reply by junegunn** [reply]
@LangLangBart Yes it is. I didn't add it to CHANGELOG then because I was unsure if `noinfo` was a good name and thought I might want to change it to something else before cutting out a release.

**Reply by LangLangBart** [reply]
> I might want to change it to something else before cutting out a release

Thanks for letting me know. I guess it's a fitting name, as I immediately associated it with the similar option for hiding the info, `--no-info`. New users might be confused between 'hidden/noinfo,' but the man page is concise and informative in its explanation.

**Comment by cmvanb** [comment]
Very informative, thank you. I will wait patiently for the official release.
